examples/client_example.py

```diff
@@ -0,0 +1,11 @@
+from generals.agents.random_agent import RandomAgent
+from generals.remote import GeneralsIOClient
+
+
+if __name__ == "__main__":
+    agent = RandomAgent()
+    with GeneralsIOClient(agent, "user_id9l") as client:
+        # register call will fail when given username is already registered
+        client.register_agent("[Bot]MyEpicUsername")
+        client.join_private_game("queueID")
```

Comment:
suggestion: maybe call this function `join_private_lobby` and the one on line 11 as ``join_game`` or something, wdyt?

Reply:
I have no preference, either works for me. Will rename it.